lib/widgets/action_sheet.dart

```diff
@@ -431,3 +505,28 @@ class ShareButton extends MessageActionSheetMenuItemButton {
     }
   }
 }
+
+class MessageActionSheetCancelButton extends StatelessWidget {
```

Comment:
nit: Let's put this right after `MessageActionSheetMenuItemButton` — there's a lot of similarities in their code, so it's helpful to see them at the same time.